Userland/Libraries/LibPDF/Reader.h

```diff
@@ -126,23 +126,21 @@ class Reader {
 #ifdef PDF_DEBUG
```

Comment:
(about the commit message)
`Foo#bar` isn't really used in C++ land, how about `Reader::dump_state`?
Also, "improve foo" sounds kinda fishy, since the description also has "this makes it a bit more readable", 
maybe you could use something like "Make Reader::dump_state a bit more readable"?

Reply:
Oops, too used to that from JVM-land